packages/ntcore-ts-client/src/lib/pubsub/topic.ts

```diff
@@ -184,7 +184,7 @@ export class NetworkTablesTopic<T extends NetworkTablesTypes> {
   /** */
 
   /**
-   * Creates a new subscriber. This should only be called by the PubSubClient.
+   * Creates a new subscriber.
```

Comment:
I removed this because it seemed like a copy/paste error from other docs in the same file. 
The readme says to use this function, so it seemed like this comment was wrong.

Since it's unrelated to the actual change, I'm happy to remove it (but figured it was good and small enough)

Reply:
Good catch